### PR TITLE
exercise deduplication

### PR DESCRIPTION
**Pull request by sstevens2**
Less to deduplicate than anticipated.  Let me know if you have any other feedback on this PR.

### CODE REVIEW

File: episodes/18-preparing.md [hunk 1]
```diff
@@ -138,7 +138,8 @@ during which we would like you to complete the following three tasks:
 The homework from this workshop includes a trial run of one of the episodes
 you have been developing in your lesson, to a real audience.
 
-Thinking about this task, what questions do you have about how you should
+After reading [the information provided about the trial run task](../learners/trial_runs.md) and thinking about this task,
```
Comment (tobyhodges) [inline episodes/18-preparing.md:141]:
```suggestion
After reading [the information provided about the trial run task](../learners/trial-runs.md) and thinking about this task, 
```

File: episodes/19-wrap-up2.md [hunk 2]
```diff
@@ -39,18 +39,7 @@ there will be an opportunity for trainees to share their experiences and reflect
...
-## What questions do you have? (5-10 min)
-
-What questions do you have about the topics and concepts covered in the training so far?
-
-After reading [the information provided about the trial run task](../learners/trial_runs.md),
-what questions do you have about this task?
-
-::::::::::::::
-
+Please provide another round of feedback and spend sometime reflecting.
```
Comment (tobyhodges) [inline episodes/19-wrap-up2.md:42]:
```suggestion
Please provide another round of feedback and spend some time reflecting.
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Minor typo fixes. Thank you @sstevens2 this is helpful. Do the episode timings add up correctly here, since the exercise was removed?

**Review by anenadic — APPROVED** [review]
Happy for this to be merged.

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**